make/targets_list.mk

```diff
@@ -81,9 +81,12 @@ UNSUPPORTED_TARGETS := \
 
 UNIFIED_TARGETS := STM32F405 \
 	STM32F411 \
+	STM32F411_CC2500 \
```

Comment:
What the reason for using ``_CC2500`` as a dedicated identifier for this? And we should document this properly such as "which target support which RF chip(s) and protocols, for ppl who using a board with that RF chip and wanna bind with what radio/RF module go flash which target".
Also, I guess there will be a series of ``STM32xxxx_SX12xx`` unified targets in the future for ELRS SPI RX right?

Reply:
This was my first idea. since this is a target wit a single RF chip i named it like this. The second on contains two RF chips. The  memory usage reduction between  the targets is relatively similar with this split. If someone has better recommendation we easily can change. The ELRS variant cold be named as you guessed. I want to show the concept.

Reply:
We also can name the targets like STM32xxxx(_CA), STM32xxxx_CC and STM32xxxx_SX? Looking forward for additional feedback.

Reply:
I'll think deeply about this.